mobile/src/main/java/com/joaquimley/avenging/ui/list/ListFragment.java

```diff
@@ -316,6 +318,12 @@ public boolean onQueryTextChange(String queryText) {
     }
 
     @Override
+    public void onDestroyView() {
+        mCharactersRecycler.setAdapter(null);
```

Comment:
Just to be sure @figengungor have you got to know a "more elegant" solution? If not I'm going to accept the PR.